docs/SRE_request_studio_content_api_addition

```diff
@@ -0,0 +1,152 @@
+
+## Changes made with introduciton of a new API
+The registrar API was added to `api-manager` in commits dated 4/17/2019 through the merge #87 commit of 6/26/2019.
+
+They entail adding the proxy description of the API to `swagger/api.yaml`
+They entail adding a registrar host variable to `scripts/aws/deploy.py`
+
+Corresponding steps were taken to add the Studio content API
+
+## Command line arguments to `deploy.py` and `flip.py`
+
+Both these repo commands take multiple command-line arguments, yet the values supplied with deployment and "go-live"
+events are not documented in the repo.
+
+### Command line arguments for `deploy.py`
+
+The call to the boto/apigateway function that carries out deployments is as follows:
+
+>    deploy_api(apig, api_id, args.swagger_filename, new_stage, {
+>        'id': args.tag,
+>        'landing_page': args.landing_page,
+>        'edxapp_host': args.edxapp_host,
+>        'discovery_host': args.catalog_host,
+>        'enterprise_host': args.enterprise_host or args.edxapp_host,
+>        'gateway_host': args.api_base_domain,
+>        'analytics_api_host': args.analytics_api_host,
+>        'registrar_host': args.registrar_host,
+>        'enterprise_catalog_host': args.enterprise_catalog_host,
+>        'studio_host': args.studio_host,
+>    })
+
+Of the command line arguments used in this call (all the `args.something` vaiables), most are integration hosts, and
+the values that were supplied at script invocation time are persisted in AWS gateway objects, under 
+"Custom domain names -> api.edx.org -> API mappings tab -> prod_black link -> Stage variables". Once there, you also 
+find persisted values for `args.id` and `args.landing_page`.
+
+This leaves only the value used for the `args.swagger_filename` unaccounted for, and that's just the pathname for
+the `swagger/api.yaml` file in this repo.
+
+### Command line arguments for `flip.py`
+
+The call to the boto/apigateway function that promotes a new stage to live status is as follows:
+
+>         update_base_path_mapping(apig, args.api_base_domain, args.next_stage)
+
+Only two command line arguments are used. The first, `args.api_base_domain` is the API Gateway custom domain name
+associated with the API in question. The second, `args.next_stage` is the name of the stage going live in support
+of that API.
+
+
+
+# <a name="runbook"></a> SRE Runbook
+
+## Preliminaries
+
+- Review this document
+- Verify that APIs on API Gateway are up and running on stage and prod as expected
+- Verify that "yellow" is the live AWS stage on the api.stage.edx.org API Gateway custom domain
+- Verify that "prod_black" is the live AWS stage on the api.edx.org API Gateway custom domain
+- Download this `api-manager` repo onto the EC2 instance hosting the API Gateway
+  - Set the working directory on a shell you'll be using to run scripts on to this repo's root directory
+
+
+## <a name="stage-runbook"></a> Deploy, activate, and test on stage
+
+### Deploy
+
```

Comment:
Can you add `stage` and `prod` as two sub-headings for the corresponding sections?